### PR TITLE
new_avg

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

**Review by iPavliv — DISMISSED** [review]

**Review by LSikyr — DISMISSED** [review]

File: main.py [hunk 1]
```diff
@@ -8,5 +9,13 @@ def run_example():
     second = input("b: ")
     print(example(first, second))
 
+
+def run_new_avg():
+    """Run new_avg function."""
+    arr = input("Array: ")
+    navg = int(input("Expected average: "))
```
Comment (khadai) [inline main.py]: here you should add try/except block for input

File: tasks/kyu7.py [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+from math import ceil
+
+
+def new_avg(arr, navg):
+    """Function find expected number that will permit to reach the average navg.
+        Args:
+            arr(list): array of numbers.
+            navg(int): expected average.
+        Returns:
+            Positive number or ValueError.
```
Comment (OlyaKhomyn) [inline tasks/kyu7.py]: Add "Raises" block to your function docstring

File: main.py (hunk 1 above)
Comment (ghost) [reply main.py]: Done.

File: tasks/kyu7.py (hunk 2 above)
Comment (ghost) [reply tasks/kyu7.py]: I did. Thank you!

**Pull request closed without merging**